### ISSUE TITLE
Variant Simultaneously Start Lost and Start Retained Consequence

### ISSUE THREAD

**Issue by DarioS** [Ensembl/ensembl-vep#727]
I see some variants that are annotated as both losing the start codon and retaining. It doesn't seem realistically possible. For example,

```
chr1:88983825_AT/A  chr1:88983826   -       ENSG00000213516 ENST00000321792 Transcript      frameshift_variant,start_lost,start_retained_variant    429     1       1       M/X     Atg/tg  -       HIGH    -       -1      -       RBMXL1  HGNC    HGNC:25073      -       -       17.52   1.81142
```

Is it a bug?

**Comment by aparton** [comment]
Hi @DarioS,

Thanks for this report. This is one that we've looked into and debated in the past within the team, and the conclusion that we've arrived at is that, while it looks unusual, it's technically correct.

We take our consequence terms and descriptions from the Sequence Ontology database, and we see the following start-codon-associated ontology terms:

start_retained: http://www.sequenceontology.org/browser/current_release/term/SO:0002019 - "A sequence variant where at least one base in the start codon is changed, but the start remains." - This is true, as while we've lost a base within the start codon, a start codon still exists.

start_lost: http://www.sequenceontology.org/browser/current_release/term/SO:0002012 - "A codon variant that changes at least one base of the canonical start codon." - The original start codon no longer exists, as such this consequence applies.

However yes, I do accept that this looks unusual, but I believe it's correct. 

Kind Regards,
Andrew
